defaults/main.yml

```diff
@@ -59,10 +59,11 @@ osbs_pruner_command_build:
 - /usr/bin/oc
 - adm
 - prune
+- "--namespace={{ osbs_namespace }}"
 - builds
 - --orphans=true
 - --confirm
 osbs_pruner_schedule_build: "0 0 * * *"
 osbs_pruner_successful_jobs: 5
 osbs_pruner_failed_jobs: 5
-osbs_pruner_clusterrole_build: system:openshift:controller:build-controller
+osbs_pruner_clusterrole_build: system:openshift:controller:build-config-change-controller
```

Comment:
Will this handle builds without build configs?

Reply:
I imagine so, permissions for handling build configs would pretty much require handling builds too

Reply:
It does.